sdk/resourcemanager/azure-resourcemanager-msi/pom.xml

```diff
@@ -76,12 +76,6 @@
       <version>1.13.11</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
why we remove this? no dependency now?

Reply:
Yes, not only is there no dependency, it also forms an infinite loop of dependencies between msi and storage.